app/javascript/vue/components/dynamic_subcomponents/RepositoryDateRangeValue.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div id="repository-date-range-value-wrapper" class="flex flex-col min-min-h-[46px] h-auto gap-[6px]">
+    <div class="font-inter text-sm font-semibold leading-5">
+      {{ colName }}
+    </div>
+    <div v-if="start_time?.formatted && end_time?.formatted"
+      class="text-sn-dark-grey font-inter text-sm font-normal leading-5 flex">
+      <div>{{ start_time.formatted }} - {{ end_time.formatted }}</div>
+    </div>
+    <div v-else class="text-sn-dark-grey font-inter text-sm font-normal leading-5">No date range</div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'RepositoryDateRangeValue',
+  data() {
+    return {
+      start_time: null,
+      end_time: null
+    }
+  },
+  props: {
+    data_type: String,
+    colId: Number,
+    colName: String,
+    colVal: Object
+  },
+  created() {
+    Object.entries(this.colVal).forEach(([key, value]) => {
+      this.$set(this, key, value)
+    })
```

Comment:
Maybe it would be better to do this more explicitly, by setting each relevant property directly.